frontend/src/concepts/pipelines/EnsureCompatiblePipelineServer.tsx

```diff
@@ -58,27 +56,14 @@ const EnsureCompatiblePipelineServer: React.FC<EnsureCompatiblePipelineServerPro
             />
             <EmptyStateBody>
               <p>
-                Rendering of this pipeline version in the UI is no longer supported, but it can
-                still be accessed via the API or OpenShift Console. To remove unsupported versions,
-                delete this project&apos;s pipeline server and create a new one.
+                This project contains v1 pipeline resources, which are no longer supported or
+                managed by OpenShift AI. To proceed, back up your pipelines data, delete the
```

Comment:
Wondering whether we should replace `OpenShift AI` here with `ODH_PRODUCT_NAME` from the env variables.